packages/preview/moderner-cv/0.1.0/lib.typ

```diff
@@ -0,0 +1,171 @@
+#import "@preview/fontawesome:0.2.1": *
+
+#let _cv-line(left, right, ..args) = {
+  set block(below: 0pt)
+  table(
+    columns: (1fr, 5fr),
+    stroke: none,
+    ..args.named(),
+    left,
+    right,
+  )
+}
+#let moderncv-blue = rgb("#3973AF")
+
+#let _header(
+  title: [],
+  subtitle: [],
+  socials: (:),
+) = {
+  let titleStack = stack(
+    dir: ttb,
+    spacing: 1em,
+    text(size: 30pt, title),
+    text(size: 20pt, subtitle),
+  )
+
+  let social(icon, link_prefix, username) = [
+    #fa-icon(icon) #link(link_prefix + username)[#username]
+  ]
+
+  let socialsList = ()
+  if "phone" in socials {
+    socialsList.push(social("phone", "tel:", socials.phone))
+  }
+  if "email" in socials {
+    socialsList.push(social("envelope", "mailto:", socials.email))
+  }
+  if "github" in socials {
+    socialsList.push(social("github", "https://github.com/", socials.github))
+  }
+  if "linkedin" in socials {
+    socialsList.push(
+      social("linkedin", "https://linkedin.com/in/", socials.linkedin),
+    )
+  }
+
+  let socialStack = stack(
+    dir: ttb,
+    spacing: 0.5em,
+    ..socialsList,
+  )
+
+  stack(
+    dir: ltr,
+    titleStack,
+    align(
+      right + top,
+      socialStack,
+    ),
+  )
+}
+
+#let moderner-cv(
+  name: [],
+  subtitle: [CV],
+  social: (:),
+  color: moderncv-blue,
+  lang: "en",
+  font: ("New Computer Modern"),
+  show_footer: true,
```

Comment:
This argument should be called `show-footer` if you want to be consistent with the rest of the package.

Reply:
Thanks, done!